.github/workflows/publish.yaml

```diff
@@ -66,7 +72,9 @@ jobs:
           --tag ${WEB_IMAGE} \
           ./web
           docker tag $WEB_IMAGE $WEB_IMAGE:${{ inputs.image-tag }}
+          if [ -n "${{ inputs.release-tag }}" ]; then docker tag $WEB_IMAGE $WEB_IMAGE:${{ inputs.release-tag }}; fi;
           docker push $WEB_IMAGE:${{ inputs.image-tag }}
+          if [ -n "${{ inputs.release-tag }}" ]; then docker push $WEB_IMAGE:${{ inputs.release-tag }}; fi;
 
   build-and-publish-api-master:
```

Comment:
Not your change; but these names are misleading. There is nothing in this workflow indicating "master".